config/condor_mapfile.osg

```diff
@@ -1,3 +1,4 @@
+GSI ".*,/[A-Za-z0-9]*/Role=[A-Za-z0-9]*/Capability=NULL.*" GSS_ASSIST_GRIDMAP
```

Comment:
I'm not sure about the `.*` at the end -- can we have `$` instead?

Reply:
Ah, also, this might not work with nested groups (`/cms/uscms`).

Reply:
I think that would possibly break FIFE jobs. IIRC, they have a ton of VOMS FQANs

Reply:
Good point about the nested groups. How deep can they go?

Reply:
Pretty deep.  I would just allow `/` in the character specification.

Reply:
I've fixed the match expression to support nested groups and limited the matches for subsequent VOMS FQANs. Are there any other characters that are supported in VO names, group names, roles, and capabilities that we need to add?